### ISSUE TITLE
Missing Textures

### ISSUE THREAD

**Issue by baumberger** [ethereumdegen/bevy_magic_fx#2]
Hi There, it seems some texture files (.png) are missing from the repository, so the examples wont compile.
The repository is very interesting, not many magic effects to be found for Bevy, which is exactly what im trying to achieve.
Any help is greatly appreciated:-)

**Comment by ethereumdegen** [comment]
Yes so I know the texture files are missing.  That just means you cant run the examples but the crate works perfectly fine.  Many of the textures and assets are licensed (not CC0) and thats why they are .gitignored  so you may have to hack the examples to make them work.. i will try and improve this.   But in any case the crate is fully functional.
